Make JetStream the default backend on prod (#14455)
kyma-project/kyma: docs/04-operation-guides/troubleshooting/evnt-02-jetstream-troubleshooting.md

2. Use the [nats CLI](https://github.com/nats-io/natscli) to check if the stream was created:

>        1. Port forward the Kyma Eventing NATS Service to localhost. Use port `4222`. Run:

Reviewer: For lines 15-31, make sure that only code is in the code block. The text should be outside of it.

          ```bash
           $ nats stream ls
           ╭────────────────────────────────────────────────────────────────────────────╮
           │                                  Streams                                   │
           ├──────┬─────────────┬─────────────────────┬──────────┬───────┬──────────────┤
           │ Name │ Description │ Created             │ Messages │ Size  │ Last Message │
           ├──────┼─────────────┼─────────────────────┼──────────┼───────┼──────────────┤
           │ sap  │             │ 2022-05-03 00:00:00 │ 0        │ 318 B │ 5.80s        │
           ╰──────┴─────────────┴─────────────────────┴──────────┴───────┴──────────────╯
           ```       

       3. If the stream exists, check the timestamp of the `Last Message` that the stream received. A recent timestamp would mean that the event was published correctly.
   
       4. Check if the consumers were created and have the expected configurations.
          ```bash
          nats consumer info
          ```
          To correlate the consumer to the Subscription and the specific event type, check the `description` field of the consumer.

>        5. If the PVC storage is fully consumed and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.

Reviewer: ```suggestion
       5. If the PVC storage is fully consumed and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which removes the old messages.
```

>        5. If the PVC storage is fully consumed, and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.

Reviewer 2: ```suggestion
       5. If the PVC storage is fully consumed and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.
```
```suggestion
       5. If the PVC storage is fully consumed, and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.
```

>        5. If the PVC storage is fully consumed, and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.

Reviewer 2: ```suggestion
       5. If the PVC storage is fully consumed and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.
```
```suggestion
       5. If the PVC storage is fully consumed, and matches the stream size as shown above, the stream can no longer receive messages. Either increase the PVC storage size or set the `MaxBytes` property which will remove the old messages.
```
